Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -19,7 +19,6 @@ file that was distributed with this source code.
     <select id="{{ id }}_autocomplete_input_v4" data-sonata-select2="false"
         {%- if read_only is defined and read_only %} readonly="readonly"{% endif -%}
         {%- if disabled %} disabled="disabled"{% endif -%}
-        {%- if required %} required="required"{% endif %}
```

Comment:
The name attribute present in input type='hidden' element/
https://github.com/sonata-project/SonataAdminBundle/blob/3.x/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig#L32

If you define name attribute for select element - then this value will be sent to the server that is undesirable.